aws/resource_aws_lambda_function.go

```diff
@@ -416,19 +437,23 @@ func resourceAwsLambdaFunctionRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("runtime", function.Runtime)
 	d.Set("timeout", function.Timeout)
 	d.Set("kms_key_arn", function.KMSKeyArn)
-	d.Set("tags", tagsToMapGeneric(getFunctionOutput.Tags))
+	d.Set("source_code_hash", function.CodeSha256)
+	d.Set("source_code_size", function.CodeSize)
 
-	config := flattenLambdaVpcConfigResponse(function.VpcConfig)
-	log.Printf("[INFO] Setting Lambda %s VPC config %#v from API", d.Id(), config)
-	vpcSetErr := d.Set("vpc_config", config)
-	if vpcSetErr != nil {
-		return fmt.Errorf("Failed setting vpc_config: %s", vpcSetErr)
+	vpcConfig := flattenLambdaVpcConfigResponse(function.VpcConfig)
+	if vpcConfig != nil {
```

Comment:
This will likely break module/output usage. Why was this changed?

Reply:
I can't really recall what I was thinking at that time 🤔